### PR TITLE
Add RitAssist support page

### PR DESCRIPTION
**Pull request by wimhaanstra**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15780

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by arsaboo — DISMISSED** [review]
Looks good...can be merged when parent PR is merged

**Review by frenck — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

File: source/_components/device_tracker.ritassist.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+title: "RitAssist"
+description: "Instructions on how to use a RitAssist as a device tracker."
+date: 2018-06-15 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ritassist.png
+ha_category: Presence Detection
+ha_release: 0.74
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.ritassist.markdown]: 0.76

File: source/_components/device_tracker.ritassist.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+---
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.ritassist.markdown]: Please add the key `ha_iot_class` with the value `Cloud Polling`.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**